Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/PaymentMethodTokenizationViewModel.swift

```diff
@@ -269,7 +269,11 @@ class PaymentMethodTokenizationViewModel: NSObject, PaymentMethodTokenizationVie
         paymentMethodButton.accessibilityIdentifier = config.type.rawValue
         paymentMethodButton.clipsToBounds = true
         paymentMethodButton.heightAnchor.constraint(equalToConstant: 45).isActive = true
-        paymentMethodButton.imageEdgeInsets = UIEdgeInsets(top: -2, left: 0, bottom: 0, right: 10)
+        let imagePadding: CGFloat = 20
+        paymentMethodButton.imageEdgeInsets = UIEdgeInsets(top: -2,
```

Comment:
Is there a reason for the extra padding?

Reply:
Yes - with the original padding, on RTL Arabic we experienced overlapping (or images and text with 0 space left and literally attached)